src/index.html

```diff
@@ -7,8 +7,79 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div>
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="moyo logo"
+            class="image"
+          >
+        </a>
+      </div>
+
+      <nav>
+        <ul class="nav-list">
+          <li class="nav__item">
```

Comment:
If you want to use bem, you need to use it correctly (element can’t exist without block 🥲)

```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
```